test/common/utxobased/db/Processor.spec.ts

```diff
@@ -589,21 +589,10 @@ describe('Processor transactions tests', () => {
     expect(tx3?.ourIns[0]).to.eqls('0')
     expect(tx3?.ourAmount).to.eqls('1')
 
-    const [tx4] = await processor.fetchTransactions({ options: {} })
-    expect(tx4).not.to.be.undefined
-
-    const results = await processor.fetchTransactions({
-      options: { startDate: new Date(11_000), endDate: new Date(15_000) }
-    })
-    expect(results).to.deep.equal([])
-
-    const [tx6] = await processor.fetchTransactions({
-      options: {
-        startDate: new Date(9_000),
-        endDate: new Date(15_000)
-      }
+    const [tx4] = await processor.fetchTransactions({
+      options: { tokenId: null }
```

Comment:
Are we deprecating some internal API without specifying the deprecation?

Reply:
The core removed these features, and the removals were all documented in the release notes. Specifically, we removed all the options to `getTransactions` besides the `tokenId`, so the core is in charge of handling the pagination, searching, and other advanced query options.

Reply:
Ah, cause the processor coupled its type with the core originally, the core upgrade impacts the processor API. Got it.